src/components/RatingStars/RatingStarts.ts

```diff
@@ -0,0 +1,9 @@
+import renderHandler from './renderHandler';
+
+const RatingStars = (movieId: number) => {
+  const ratingStars = renderHandler(movieId);
+
+  return ratingStars;
```

Comment:
```suggestion
  return renderHandler(movieId);
```

으로 해도 여기선 괜찮을것 같은데 어떻게 생각하시나요?

Reply:
기존 코드는 불필요한 할당 맞는것 같습니다 감사합니다!